dot/state/block.go

```diff
@@ -250,6 +251,37 @@ func (bs *BlockState) GetHashByNumber(num uint) (common.Hash, error) {
 	return common.NewHash(bh), nil
 }
 
+// GetHashesByNumber returns the block hashes with the given number
+func (bs *BlockState) GetHashesByNumber(num uint) ([]common.Hash, error) {
+	blockHashes := []common.Hash{}
+	block, err := bs.GetBlockByNumber(num)
+	if err == nil {
+		blockHashes = bs.bt.GetAllBlocksAtNumber(block.Header.ParentHash)
+	}
```

Comment:
Seems odd that we don't care if this return an error.  Eventually this will call `BlockTree.GetHashByNumber` which returns a number of different sentinel errors.  Should we proceed in every error case?

Reply:
fixed it